internal/driver/driver.go

```diff
@@ -478,10 +492,12 @@ func (d *Driver) Stop(force bool) error {
 func (d *Driver) getPathName(device *Device, queryParams url.Values) (string, error) {
 	var videoPath string
 	pathIndex := queryParams.Get(PathIndex)
-	if len(pathIndex) == 0 {
-		// currently defaults to using the first available stream
+	streamFormat := queryParams.Get(StreamFormat)
+	if len(pathIndex) == 0 && len(streamFormat) == 0 { // most common case
```

Comment:
if these are strings, just use `if pathIndex == ""`, etc.

this is used a lot it looks like